oj/cf.py

```diff
@@ -0,0 +1,22 @@
+import urllib.request
```

Comment:
I think we can use `requests` module in all the OJ fetcher. (Maybe I should add a `contribute.md`.)